packages/next/build/webpack/loaders/next-app-loader.ts

```diff
@@ -108,7 +109,7 @@ async function createTreeCodeFromPath({
               return `${
                 file === FILE_TYPES.layout
                   ? `layoutOrPagePath: ${JSON.stringify(filePath)},`
-                  : ''
+                  : `filePath: ${JSON.stringify(filePath)},`
```

Comment:
This makes the bundle slightly larger, but it's optimizable in the future together with `layoutOrPagePath`. Maybe a mapping manifest in somewhere else.

Reply:
This outputs duplicate `filePath` keys, will push up a commit fixing this.